quetz/config.py

```diff
@@ -71,6 +71,7 @@ class Config:
             "general",
             [
                 ConfigEntry("package_unpack_threads", int, 1),
+                ConfigEntry("package_queue_size", int, 1),
```

Comment:
hmmm ... this is pretty specific to the reindexing task ... couldn't this be an arguement to the task, actually?

Reply:
Yes, I can turn it into an argument for the task.